bindgen/clang.rs

```diff
@@ -70,6 +71,16 @@ impl fmt::Debug for Cursor {
     }
 }
 
+trait ToUsize {
+    fn to_usize(self) -> usize;
+}
+
+impl ToUsize for c_uint {
+    fn to_usize(self) -> usize {
+        self.try_into().expect("`c_uint` does not fit in `usize`")
+    }
+}
+
```

Comment:
I don't think this needs to be a special trait. For areas where you don't need special handling, `.try_into().unwrap()` is reasonably terse and already cites a `TryFromIntError`. And in other areas, you probably want to `.try_into().ok_or(SomeError)?` instead

Reply:
Personally, I don't mind either way, I only added this trait to address

> seems less legible

Reply:
> Personally, I don't mind either way, I only added this trait to address
> 
> > seems less legible

Ah, maybe that was in reference to not having the type? If so, using TryFrom rather than TryInto (`u32::try_from(val).unwrap()`) could make intent more clear. I don't think adding a new trait just to wrap this makes things any more clear.